src/Microsoft.Extensions.WebEncoders.Core/UrlEncoderWrapper.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text.Encodings.Web;
+
+namespace Microsoft.Extensions.WebEncoders
+{
+    public class UrlEncoderWrapper : IUrlEncoder
+    {
+        private UrlEncoder _encoder;
+
+        public UrlEncoderWrapper()
+            : this(UrlEncoder.Default)
+        {
+        }
+
+        public UrlEncoderWrapper(UrlEncoder encoder)
+        {
+            _encoder = encoder;
+        }
+
+        public UrlEncoderWrapper(TextEncoderSettings settings)
+        {
+            _encoder = UrlEncoder.Create(settings);
+        }
+
+        public static IUrlEncoder Default { get; } = new UrlEncoderWrapper(UrlEncoder.Default);
```

Comment:
This default is for plugging into places that take the interface, see https://github.com/aspnet/HttpAbstractions/pull/449/files#diff-af79926afbc0631c7212bff6ed282bc0R208